tests/common.go

```diff
@@ -4867,6 +4860,10 @@ func StartTorpedoTest(testName, testDescription string, tags map[string]string,
 	tags["pureVolume"] = fmt.Sprintf("%t", Inst().PureVolumes)
 	tags["pureSANType"] = Inst().PureSANType
 	dash.TestCaseBegin(testName, testDescription, strconv.Itoa(testRepoID), tags)
+	if TestRailSetupSuccessful && testRepoID != 0 {
+		RunIdForSuite = testrailuttils.AddRunsToMilestone(testRepoID)
+		CurrentTestRailTestCaseId = testRepoID
```

Comment:
will this not be prob for PX cases. please check once

Reply:
It is not disruptive operation. It will not fail for PX tests

Reply:
`AddRunsToMilestone` will just check if a run needs to be added or if it already exists. If it already exists it doesn't do anything.

Reply:
it will not fail but Im thinking it might add duplicate entries in testrail